updated chi guidelines

steveharoz opened the pull request:
Blog post for chi guidelines.

Also, I made some small theme adjustments that just affect spacing to help make all the quotes and section breaks more readable. It doesn't appear to adversely affect the previous posts.

steveharoz committed:
updated chi guidelines

steveharoz committed:
spacing in theme

steveharoz commented:
Test site: http://steveharoz.com/test/transstats/2019/08/01/updates-to-chi-submission-and-reviewing-guides/

steveharoz committed:
dropping heading spacing adjustment

mjskay commented on themes/hugo-lithium/static/css/main.css:
```diff
@@ -366,3 +366,14 @@ code {
+
+/* bring quotes coser to previous paragraph */
+blockquote {
+  margin-block-start: -1em;
```
This feels a little too close now to my eye... also, given that the inter-para spacing is set by pixels and not ems, this is inconsistent when you resize. What about something like `margin-block-start: -10px;` ?

steveharoz replied:
The theme mixes em and px, but yeah the main spacing for paragraphs is `25px`. I've tweaked it a bit so it's clear that the bullets and blockquotes are connected to the preceding paragraph but aren't smashed into it.

mjskay replied:
it still looks a little close to me... if your eye says -15px and mine says -10px, maybe we compromise on -12.5px? That also happens to be half the inter-para spacing, so maybe that's good?

(also you set the bullet spacing to be -19px, which definitely looks too close to me. Whatever we set this to I would make them both the same)

steveharoz replied:
OK. I set both to -12.5px

steveharoz replied:
and preview site's updated

mjskay commented on themes/hugo-lithium/static/css/main.css:
```diff
@@ -366,3 +366,14 @@ code {
+
+/* bring bullets closer to previous paragraph */
+ul, ol { 
+  margin-top: -1em;
```
Same comment as above. I would change to `margin-top: -10px;`

I would also add `margin-bottom: 25px;`, looks like the margin after is currently not consistent with paragraphs and probably should be.

steveharoz replied:
combined this with the above style for consistency

mjskay commented on content/blog/2019-06-27-updates-to-the-chi-submission-and-reviewing-guides.Rmd:
```diff
@@ -0,0 +1,81 @@
+
+New or updated text in the reviewing guide:
+
+> Lack of transparency in the way research results are reported can be a ground to doubt the contribution. See the “Transparency” section in “Guide to a Successful Submission” for a discussion of transparency in different contribution types.
```
I would consider italicizing the text in the block quotes to make them a little more distinct and to follow the style in the other blog post.

steveharoz replied:
Because there is stylizing in the guideline that we're quoting (some text is bold), I was trying to avoid stylizing the quotes. I'll try a couple options to make the blockquotes stand out a bit.

mjskay replied:
the color approach looks good to me

mjskay commented on content/blog/2019-06-27-updates-to-the-chi-submission-and-reviewing-guides.Rmd:
```diff
@@ -0,0 +1,81 @@
+tags: [chi]
+---
+
+Over the past year, members of [Transparent Statistics in HCI](https://transparentstatistics.org/) have worked with people across various disciplines of the HCI community and chairs of the conference (full contributor list below) to update the CHI guides for a successful submission and for reviewing. These new instructions aim to increase the transparency and openness of research, including:
```
Maybe "... aim to increase the transparency and openness of _several facets of_ research, including:"

Without come clarification the list feels a little awkward since i'm not sure exactly what is about to be listed (types of research? types of instructions to improve research?)

mjskay commented on content/blog/2019-06-27-updates-to-the-chi-submission-and-reviewing-guides.Rmd:
```diff
@@ -0,0 +1,81 @@
+* materials created or used
+* data collected
+
+The availability of these facets of research can enable reviewers and readers to better assess HCI research, facilitate industry to build upon and apply HCI research, and help HCI researchers to more efficiently use past research to make new advances.
```
just "availability" is a little vague. Maybe "Increasing the availability..." ? Or "Making these facets of research more available"?

mjskay commented on content/blog/2019-06-27-updates-to-the-chi-submission-and-reviewing-guides.Rmd:
```diff
@@ -0,0 +1,81 @@
+
+> CHI papers should strive for research transparency regardless of the contribution type and methodology. Different contribution types, (e.g. technical contributions, quantitative studies, and qualitative studies) use different criteria for assessing transparency.  
+> […]  
+> **Research transparency is of utmost importance in a CHI paper. It allows reviewers to understand and assess submitted work thoroughly, and it allows members of the research community to understand, analyze, and build upon the work in published CHI papers. As such transparency is taken into account very seriously in the review process.**
```
Is this bold in the changed guideline? If not I'm not sure why it should be bold here...

steveharoz replied:
yes, it's bold in the guideline. That's why I'm hesitant to use font styling for the block quotes.

mjskay commented on content/blog/2019-06-27-updates-to-the-chi-submission-and-reviewing-guides.Rmd:
```diff
@@ -0,0 +1,81 @@
+
+## Quantitative and qualitative methods
+
+Regardless of research methodology (quantitative or qualitative; empirical or engineering) CHI encourage transparent research practices. While the specific methods may differ in what materials and data are created, reviewers can expect all facets of the research to be transparently reported.
```
"...CHI encourage**s**..." (missing s)

mjskay reviewed (changes requested):
Overall looks good, thanks Steve! I've added some mostly minor comments.

steveharoz committed:
chi guidelines - clarifications and typo

steveharoz committed:
blockquote and list tweaks

steveharoz commented:
Updated the preview - http://steveharoz.com/test/transstats/2019/08/01/updates-to-chi-submission-and-reviewing-guides/
* Made suggested text changes.
* Tweaked the spacing and added a slight color change for the blockquotes, so they stand out without having bold+italic text

steveharoz committed:
spacing adjustment

mjskay commented:
Looks good to me, thanks!

steveharoz commented:
Chat approved earlier via slack. Thanks to both for comments!
